src/core/Akka/IO/Inet.cs

```diff
@@ -221,6 +222,39 @@ public override void AfterConnect(Socket s)
                     //TODO: What is the .NET equivalent
                 }
             }
+
+            public class ByteBufferPoolSize : SocketOption
+            {
+                public int ByteBufferPoolSizeBytes { get; }
+
+                public ByteBufferPoolSize(int sizeBytes)
+                {
+                    ByteBufferPoolSizeBytes = sizeBytes;
+                }
+            }
+
+            public class TlsConnectionOption : SocketOption
+            {
+                public X509Certificate2 Certificate { get;  }
+                public bool SuppressValidation { get; }
+                public bool ClientAsServer { get; set; }
+
+                public TlsConnectionOption(X509Certificate2 cert, bool suppressValidation)
+                {
+                    Certificate = cert;
+                    SuppressValidation = suppressValidation;
+                }
+            }
+
+            public class WorkerDispatcher : SocketOption
```

Comment:
Similar to ByteBuffer setting, this lets a User have a TCP Connection run on a specific Dispatcher instead of Akka.IO Defaults in HOCON